apps/DDnet/install

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#Dependencies
+install_packages build-essential cmake git libcurl4-openssl-dev libssl-dev libfreetype6-dev libglew-dev libnotify-dev libogg-dev libopus-dev libopusfile-dev libpnglite-dev libsdl2-dev libsqlite3-dev libwavpack-dev python google-mock libx264-dev || exit 1
+
+package_available glslang-tools
+if [[ $? == "0" ]]; then
+  install_packages glslang-tools glslang-dev || error "Failed to install dependencies"
```

Comment:
This seems to be compiling fine for me with glslang-tools installed but glslang-dev not installed.
Once it's done I'll check if it runs.